common/credCacheInternal_linux.go

```diff
@@ -54,22 +53,6 @@ func (c *CredCacheInternalIntegration) HasCachedToken() (bool, error) {
 	return has, err
 }
 
-// RemoveCachedToken deletes the cached token.
-func (c *CredCacheInternalIntegration) RemoveCachedToken() error {
-	c.lock.Lock()
-	err := c.removeCachedTokenInternal()
-	c.lock.Unlock()
-	return err
-}
-
-// SaveToken saves an oauth token.
-func (c *CredCacheInternalIntegration) SaveToken(token OAuthTokenInfo) error {
-	c.lock.Lock()
-	err := c.saveTokenInternal(token)
-	c.lock.Unlock()
-	return err
-}
-
 // LoadToken gets the cached oauth token.
 func (c *CredCacheInternalIntegration) LoadToken() (*OAuthTokenInfo, error) {
```

Comment:
Do we even need any credCache code in AzCopy anymore? Could we just remove all of these files?

Reply:
Removed in the latest commit

Reply:
Why are we unable to? What piece of code in AzCopy still relies on this (other than cred cache)?